pkgs/by-name/su/sunvox/package.nix

```diff
@@ -58,6 +60,21 @@ stdenv.mkDerivation (finalAttrs: {
     libjack2
   ];
 
+  desktopItems = lib.optionals stdenv.hostPlatform.isLinux [
+    (makeDesktopItem {
+      name = "sunvox";
+      exec = "sunvox";
```

Comment:
Note that there are multiple binaries to pick between different rendering backends, some of which are also only included on specific platforms like aarch64-linux.

Prolly fine to just start with the main one for now though.